src/CaptureFile/include/CaptureFile/CaptrureSectionInputStream.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_FILE_MAIN_SECTION_INPUT_STREAM_H_
+#define CAPTURE_FILE_MAIN_SECTION_INPUT_STREAM_H_
+
+#include "OrbitBase/Result.h"
+#include "capture.pb.h"
+
+namespace orbit_capture_file {
+
+class CaptureSectionInputStream {
```

Comment:
Could you add a super brief comment for this class?

Reply:
done.